src/core/constants.js

```diff
@@ -696,90 +696,90 @@ export const BOTTOM = 'bottom';
  * @final
  */
 export const BASELINE = 'alphabetic';
-/**
- * @typedef {unique symbol} PATH
- * @property {PATH} PATH
- * @final
- */
-export const PATH = Symbol('path');
 /**
  * @typedef {unique symbol} EMPTY_PATH
  * @property {EMPTY_PATH} EMPTY_PATH
  * @final
  */
 export const EMPTY_PATH = Symbol('empty_path');
 /**
- * @typedef {0x0000} POINTS
+ * @typedef {unique symbol} PATH
+ * @property {PATH} PATH
+ * @final
+ */
+export const PATH = Symbol('path');
+/**
+ * @typedef {unique symbol} POINTS
  * @property {POINTS} POINTS
  * @final
  */
-export const POINTS = 0x0000;
```

Comment:
Heads up, currently changing these will break WebGL because there's still an assumption internally that p5's POINTS is equal to the webgl `gl.POINTS` constant. If possible, it'd be better to avoid refactoring all of this right now

Reply:
Okay, no prob! I went ahead and reimplemented all the custom shape constants as hex numbers, in keeping with the old way of doing things, and I refactored `PrimitiveShapeCreators` accordingly. Tests are passing now. (I left a to-do comment in the code to switch to Symbol constants whenever the time is right.)